dart/biomechanics/MarkerFitter.cpp

```diff
@@ -3772,6 +3772,7 @@ MarkerInitialization MarkerFitter::getInitialization(
   if (params.dontRescaleBodies && params.groupScales.size() > 0)
   {
     result.groupScales = params.groupScales;
+    mSkeleton->setGroupScales(result.groupScales);
```

Comment:
Was this just missing before, or is this now needed for adding in the anthropometric prior term?

Reply:
In theory, every method should set the skeleton to the right body scales before evaluating gradients or Jacobians. Just to be on the safe side, I just set it on the skeleton here too, in case anywhere is forgetting to set it.

This is a "defensive" change rather than a bug fix per se.